crates/cheatcodes/src/utils.rs

```diff
@@ -181,3 +167,134 @@ impl Cheatcode for copyStorageCall {
         Ok(Default::default())
     }
 }
+
+impl Cheatcode for arbitraryUint_0Call {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        arbitrary_uint(state, None, None)
+    }
+}
+
+impl Cheatcode for arbitraryUint_1Call {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self { min, max } = *self;
+        arbitrary_uint(state, None, Some((min, max)))
+    }
+}
+
+impl Cheatcode for arbitraryUint_2Call {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self { bits } = *self;
+        arbitrary_uint(state, Some(bits), None)
+    }
+}
+
+impl Cheatcode for arbitraryInt_0Call {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        arbitrary_int(state, None)
+    }
+}
+
+impl Cheatcode for arbitraryInt_1Call {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self { bits } = *self;
+        arbitrary_int(state, Some(bits))
+    }
+}
+
+impl Cheatcode for arbitraryAddressCall {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        arbitrary_address(state)
+    }
+}
+
+impl Cheatcode for arbitraryBoolCall {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        Ok(DynSolValue::type_strategy(&DynSolType::Bool)
+            .new_tree(state.test_runner())
+            .unwrap()
+            .current()
+            .abi_encode())
+    }
+}
+
+impl Cheatcode for arbitraryBytesCall {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self { len } = *self;
+        let mut val = DynSolValue::type_strategy(&DynSolType::Bytes)
+            .new_tree(state.test_runner())
+            .unwrap()
+            .current()
+            .as_bytes()
+            .unwrap_or_default()
+            .to_vec();
+        let required_len = len.to::<usize>();
```

Comment:
Same, this needs an `ensure!`

Reply:
added in https://github.com/foundry-rs/foundry/pull/8882/commits/1c2e688c7e76bc79801e052a49239ea3061793ce + test. Put limit to usize max which doesn't make really sense as for big lengths it fails with out of gas, but leaving up to user to figure out limit